twitter.html

```diff
@@ -0,0 +1,568 @@
+                    </svg></div>
+            </div>
+            <div class="post">
+                <div class="picture">
+                    <img src="/pngegg.png" alt="" class="harry">
+                </div>
+                <div class="post-content">
+                    <div class="post-text">
+                        <input type="text" placeholder="What's happening?">
```

Comment:
I was trying to figure out how to make the what's happening disappear on mine, This has shown me how!